src/test/resources/net/sf/jsqlparser/statement/select/oracle-tests/merge04.sql

```diff
@@ -26,4 +26,5 @@ update set mm.inserts = mm.inserts + v.inserts, mm.updates = mm.updates + v.upda
 mm.flags = mm.flags + v.flags - bitand(mm.flags,v.flags) , mm.drop_segments = mm.drop_segments + v.drop_segments
```

Comment:
Is this supposed to fail?
The query parses/formats [well with JSQLFormatter.](http://jsqlformatter.manticore-projects.com/jsqlformatter/demo.html?args=-c%20LIUQSg4iAED0BUBqaARAmgOQILAJIGEB9AZRwAUAZXDCACmGGgBYBKVTHAk8qmwkYgBVC+FBnrAWAKGiy58+emx4ipYJWp1Grdsq5qNfAcNHjJ0eLGjVBAeWgBnAJ4OAdAFsA9gDtCXgCYOhACGADahACTQ7u4ycgCqxJrQtNDEIBQg+IKyKACmAMae-nmpEABOngCuAA4Alt4A5oS4-qkALjWuAEaeAMQANNDtDl01nt0AVoPRrhPT0NIKyyurskMAzEOdPf1xawcrQwCM26Ou41N9+4e367PzfYvQjzd3ckPEVe6p7q4NDjy5RGzwBQJGb3e0E+31+rlq-mC7TyDmeCKRKMh7xhP1mJVCeWRqLY+MJmKhCiG+SKJVSX1xACE6u1gt42rMAGahYKNBwnZ5LClQoYABlFWKFq35bGQ1OKpTSsOgTJZbLhXJ5fOgACYBRLJQaPtAxcb9YaDUNdYLzctZYV5XSlSrWey-hreUMmHqbZLReKfULPc93Q4zQccXD-JUaoRAY13HlvCCSdHY3l44mIbcAGJgWyMZxuLy+AIOKKxClDQuuFndIruGrBYFRTphymOFw14LdBxVbqN5vDUZtgDqAAlwDA-o9oABeIddV4UrAYFDDRdXOcLi5Lw4QPPxMjKtDQMCecK1DpdXozEZjR5Daeb61rMdYABqyQq1XqTRa7J2G8znvK5HzmZ9oAAHmgAB2NtbDAFBwGPeRjjbW5LXQw5NmeAA3CVbAwFJiJicCFnnXCyKeNhpHHEAiOALBBHwCc10ECcMBuQ8UCYmB0hyaI-jBYFUXnUjhJBZBKIk0NK0E+EakRIkt1I9FlKkhSlPJYV5NJZSxL+PSUWgDSjNknTSJDFS3W5XkTOgSirIAWmVZkXV+GzNSGRzbOJLD7lIqNPBjOMEyTUTdNTULM1RUyovTMKITooiMFsHJGOY1joHY+ibmodIwByd8sAoeIBGIyjd19Bz-m8QERP89tKLU7TqsoszGvkKsXC0zqjR8zU+vudr4ozcLFikABuIA)